docs/how-to-guides/format-api.md

```diff
@@ -78,7 +78,7 @@ registerFormatType( 'my-custom-format/sample-output', {
 } );
 ```
 
-Let's check that everything is working as expected. Build and reload and then select a text block. Confirm the new button was added to the format toolbar.
+Let's check that everything is working as expected. Build and reload and then select a paragraph block. Confirm the new button was added to the format toolbar.
```

Comment:
How do you feel about changing this from paragraph block to "any block containing text like for example the paragraph block"

I know it's longer but I want to avoid that people thing it just applies to the paragraph block :)

Reply:
Fixed by 4d729c0